rio/src/RIO/Prelude/Reexports.hs

```diff
@@ -299,3 +305,6 @@ import qualified GHC.Stack
 import qualified Prelude
 import qualified System.Exit
 import qualified Text.Read
+
+yieldThread :: MonadIO m => m ()
+yieldThread = UnliftIO.Concurrent.yield
```

Comment:
Should this be inlined?

Reply:
Good catch, I added the inline pragma now